### ISSUE TITLE
F# project doesn't build with preview1

### ISSUE THREAD

**Issue by tmds** [dotnet/cli#6574]
## Steps to reproduce

```
dotnet new console -lang f#
dotnet build
```

## Expected  behavior

Project builds.

## Actual behavior

```
Microsoft (R) Build Engine version 15.3.117.23532
Copyright (C) Microsoft Corporation. All rights reserved.

/home/tmds/.nuget/packages/fsharp.net.sdk/1.0.4/build/FSharp.NET.Core.Sdk.targets(170,9): error MSB6006: "dotnet" exited with code 134. [/tmp/dotnet-latest/fsapp/fsapp.fsproj]

Build FAILED.
```

## Environment data

```
.NET Command Line Tools (2.0.0-preview1-005977)

Product Information:
 Version:            2.0.0-preview1-005977
 Commit SHA-1 hash:  414cab8a0b

Runtime Environment:
 OS Name:     fedora
 OS Version:  24
 OS Platform: Linux
 RID:         fedora.24-x64
 Base Path:   /tmp/dotnet-latest/sdk/2.0.0-preview1-005977/

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0-preview1-002111-00
  Build    : 1ff021936263d492539399688f46fd3827169983

```

**Comment by livarcocc** [comment]
@KevinRansom @enricosada this is because the current fsharp compiler still targets 1.1 and the 2.0 CLI does not carry that runtime anymore.

**Comment by enricosada** [comment]
@livarcocc i know. atm fsharp compiler target 1.0 not 1.1 (so is not bundled in in sdk 1.0 for some OS either).

@tmds the issue tracking this is https://github.com/dotnet/netcorecli-fsc/issues/108 , i'll try to update wiki later, but no workaround atm for this preview (when fsc is ready i'll fix 2.0-preview1 with a new fsharp.net.sdk release, i hope to have it soon)
The sdk 1.0 works ok atm.

**Comment by livarcocc** [comment]
Closing this at it is already being tracked in https://github.com/dotnet/netcorecli-fsc/issues/108.
